src/index.ts

```diff
@@ -63,7 +64,21 @@ export function validationMetadataArrayToSchemas(
             isExcluded(propMeta, options) ||
             isExcluded({ ...propMeta, target }, options)
           )
-      )
+      ).map((propMeta) => {
+            /**
+             * Retrieves all properties that have the Expose decorator from class-transformer
+             * and remaps the property names to the names exposed by the Expose decorator.
+             */
+          const ctMetadata = userOptions?.classTransformerMetadataStorage?.getExposedMetadatas(propMeta.target as any)
+
+          const ctMetaForField = ctMetadata?.find((meta: ExposeMetadata) => meta.propertyName == propMeta.propertyName)
+
+          if (ctMetaForField) {
+              propMeta.propertyName = ctMetaForField.options.name ?? propMeta.propertyName
+              return propMeta
+          }
+          return propMeta
```

Comment:
```suggestion
          if (ctMetaForField?.options.name) {
              propMeta.propertyName = ctMetaForField.options.name;
          }

          return propMeta
```
A little more concise.